src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurCashAdvancePdpFeedFileServiceImpl.java

```diff
@@ -172,6 +185,68 @@ private PdpFeedTrailerEntry buildPdpFeedTrailerEntry(int totalDetailRecordCount,
         return trailer;
     }
 
+    private void updateReportDataPdpFeedFileCountAndAmountTotals(ConcurRequestExtractBatchReportData reportData, ConcurRequestExtractRequestDetailFileLine detailFileLine) {
+        int count = reportData.getCashAdvancesProcessedInPdp().getRecordCount();
+        count++;
+        reportData.getCashAdvancesProcessedInPdp().setRecordCount(count);
+        KualiDecimal amount = reportData.getCashAdvancesProcessedInPdp().getDollarAmount();
+        reportData.getCashAdvancesProcessedInPdp().setDollarAmount(amount.add(detailFileLine.getRequestAmount()));
+    }
+
+    private void updateReportDataClonedCashAdvanceCountAndAmountTotals(ConcurRequestExtractBatchReportData reportData, ConcurRequestExtractRequestDetailFileLine detailFileLine) {
+        int count = reportData.getClonedCashAdvanceRequests().getRecordCount();
+        count++;
+        reportData.getClonedCashAdvanceRequests().setRecordCount(count);
+        KualiDecimal amount = reportData.getClonedCashAdvanceRequests().getDollarAmount();
```

Comment:
I think it might be cleaner to add a new method in the ConcurBatchReportSummaryItem class that adds the input amount to the DollarAmount.